api/src/main/java/org/openmrs/module/fhir2/api/FhirTaskService.java

```diff
@@ -32,4 +32,12 @@
 	 */
 	public Task saveTask(Task task);
 	
+	/**
+	 * Save task to the DB
+	 * 
+	 * @param task the task to save
+	 * @return the saved task
+	 */
+	public Task saveTask(String uuid, Task task);
```

Comment:
We don't need the `public` modifier in an interface. Can you also change this for the other two methods here?